pkg/attestation/crafter/api/attestation/v1/crafting_state.proto

```diff
@@ -58,6 +58,8 @@ message Attestation {
     message KeyVal {
       string id = 1 [(buf.validate.field).string.min_len = 1];
       string value = 2 [(buf.validate.field).string.min_len = 1];
+
+      string digest = 3;
```

Comment:
should we add validation too? ` [(buf.validate.field).string.min_len = 1];`?

Reply:
I was thinking that, but what happens with in-progress attestations?

Reply:
Added